src/Compilers/VisualBasic/Portable/Parser/ParseConditional.vb

```diff
@@ -8,10 +8,12 @@
 Imports System.Globalization
 Imports Microsoft.CodeAnalysis.Syntax.InternalSyntax
 Imports InternalSyntaxFactory = Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax.SyntaxFactory
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax.FeatureUtils
```

Comment:
`Imports` is not required here or in other files containing `InternalSyntax` types below.